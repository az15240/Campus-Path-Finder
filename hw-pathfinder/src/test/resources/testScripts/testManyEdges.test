# Test a special case where there are multiple edges connecting two nodes

CreateGraph A
AddNode A n1
AddNode A n2
AddNode A n3
AddEdge A n1 n2 1
AddEdge A n1 n2 2
AddEdge A n1 n2 3
AddEdge A n2 n3 4
AddEdge A n2 n3 5
AddEdge A n1 n3 10

ListChildren A n1
ListChildren A n2
ListChildren A n3

FindPath A n1 n3