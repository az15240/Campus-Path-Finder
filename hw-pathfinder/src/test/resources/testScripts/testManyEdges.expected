# Test a special case where there are multiple edges connecting two nodes

created graph A
added node n1 to A
added node n2 to A
added node n3 to A
added edge 1.000 from n1 to n2 in A
added edge 2.000 from n1 to n2 in A
added edge 3.000 from n1 to n2 in A
added edge 4.000 from n2 to n3 in A
added edge 5.000 from n2 to n3 in A
added edge 10.000 from n1 to n3 in A

the children of n1 in A are: n2(1.000) n2(2.000) n2(3.000) n3(10.000)
the children of n2 in A are: n3(4.000) n3(5.000)
the children of n3 in A are:

path from n1 to n3:
n1 to n2 with weight 1.000
n2 to n3 with weight 4.000
total cost: 5.000